src/compounds/side-nav/src/index.tsx

```diff
@@ -0,0 +1,111 @@
+/** @jsx jsx */
+
+import * as React from 'react'
+import { jsx, useThemeUI } from 'theme-ui'
+import Accordion from '@uswitch/trustyle.accordion'
+
+interface Link {
+  text: string
+  url: string
+  isActive?: boolean
+}
+
+interface LinkGroup {
+  title: string
+  links: Link[]
+}
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+  internalLinks: LinkGroup
+  additionalLinks?: LinkGroup[]
+}
+
+const SideNav: React.FC<Props> = ({ internalLinks, additionalLinks = [] }) => {
+  const { theme: { sideNav: sideNavTheme = {} } = {} }: any = useThemeUI()
```

Comment:
As far as I understand, this isn't the best way to do this - best to use `variant` instead: https://theme-ui.com/guides/variants/#themeable-layout-components

Reply:
ah yeah good shout